elements/pl-figure/pl-figure.py

```diff
@@ -8,11 +8,12 @@
 TYPE_DEFAULT = 'static'
 DIRECTORY_DEFAULT = 'clientFilesQuestion'
 INLINE_DEFAULT = False
+ALT_TEXT_DEFAULT = 'PrairieLearn Figure'
```

Comment:
```suggestion
ALT_TEXT_DEFAULT = 'PrairieLearn logo'
```
I suggest being as specific as possible. It's generally recommended to avoid generic terms like "image" and "figure" in alt text.

Reply:
I thinking that we should default to empty alt text, assuming that the image is described by the surrounding text as is common in many questions. My understanding is that when the image is described by the surrounding text, we should not specify alt text?

Reply:
Would you prefer for the alt text field in the img tog to be `alt=""` or would you prefer for there just to be no alt field whatsoever?

Reply:
According to the link that @tdy posted, all images should have an `alt` attribute and we are supposed so specify `alt=""` if it should be empty. So let's always add it and just set `alt=""`.

Reply:
done